### PR TITLE
Add circle yml

### PR DESCRIPTION
**Pull request by tanukiti1987**
Let's run test on Circle CI, yey!

Can you check it? @oishi-kenko/engineers

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+dependencies:
+  cache_directories:
+    - 'vendor/bundle'
+  override:
+    - bundle -j8 --path=vendor/bundle
+
+database:
+  override:
+    - bundle exec rake db:create RAILS_ENV=test
+    - bundle exec rake db:migrate RAILS_ENV=test
```
Comment (manemone) [inline circle.yml:14]: [IMO] I think this line may not be necessary while this code automatically do the same thing on every run.

https://github.com/oishi-kenko/katgut/blob/master/spec/rails_helper.rb#L11

**Comment by manemone** [comment]
Great PR! Really...

LGTM with a IMO.

I turned on the notification for a pull request submission after I commented the `circle.yml`.
Could you take a look at the comment before you merge the PR?

https://github.com/oishi-kenko/katgut/pull/10#discussion_r93174516

File: circle.yml (hunk 1 above)
Comment (manemone) [reply circle.yml:14]: Oops, my mistake. Please forget about it.

Comment (tanukiti1987) [reply circle.yml:14]: 😇

**Pull request merged**